Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -170,7 +178,14 @@ YUI.add('ez-locationviewviewservice', function (Y) {
                     'done',
                     5
                 );
-
+                /**
+                 * Fired when the content is moved
+                 *
+                 * @event movedContent
+                 * @param {eZ.Location} location
+                 * @param {eZ.LocationId} parentLocationId
```

Comment:
should be `@param {String} oldParentLocationId` -and should be changed in the implementation as well)